hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -171,7 +171,7 @@ public static HoodieWriteConfig getHoodieClientConfig(
             .withMergeAllowDuplicateOnInserts(OptionsResolver.insertClustering(conf))
             .withClusteringConfig(
                 HoodieClusteringConfig.newBuilder()
-                    .withAsyncClustering(conf.getBoolean(FlinkOptions.CLUSTERING_ASYNC_ENABLED))
+                    .withAsyncClustering(conf.getBoolean(FlinkOptions.CLUSTERING_ASYNC_ENABLED) || conf.getBoolean(FlinkOptions.CLUSTERING_SCHEDULE_ENABLED))
```

Comment:
```java
// mainly for clustering scheduling
withAsyncClustering(conf.getBoolean(FlinkOptions.CLUSTERING_SCHEDULE_ENABLED))
```

Config only as scheduling option should work.

Reply:
Right, changed to scheduling option only, thanks